ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/OpenSSL/tasks/main.yml

```diff
@@ -4,57 +4,57 @@
 ###########
 - name: Check if OpenSSL 32bit VS2013 installed
   win_stat:
-    path: C:\openjdk\OpenSSL-1.1.1e-x86_32-VS2013
+    path: C:\openjdk\openssl-1.1.1g-x86_32-VS2013
```

Comment:
Is there any particular reason you're changing `OpenSSL` to `openssl` ?

Reply:
No particular reason, does it need to be `OpenSSL`?

Reply:
Yeah, the build scripts will look for them at the specific name: https://github.com/AdoptOpenJDK/openjdk-build/blob/3b9f2ec5883e52b6474d5ac1e9e6772dbe1dcb5b/build-farm/platform-specific-configurations/windows.sh#L78

I'm not entirely sure if the capitalisation is *required* , but if we don't have to change it, I don't think we should :-)

Reply:
Good call will fix up now